codecov.yaml

```diff
@@ -9,3 +9,7 @@ comment:
   require_base: no
   require_head: no
   after_n_builds: 1
+
+ignore:
+  - "**/testing/mock_*.go"
+  - "**/*generate*.go"
```

Comment:
Just thought of "pkg/client/" is generated too.

Reply:
Updated.

Reply:
could we add `**.pb.go` as well? otherwise a file like `pkg/apis/cni/v1beta1/cni.pb.go` would be included in the report.

We may want to exclude the `third_party` directory as well, but I'll leave it up to you.

Reply:
Updated.